DigitalLearningSolutions.Web/ViewModels/Login/ChooseACentreViewModel.cs

```diff
@@ -5,13 +5,22 @@
 
     public class ChooseACentreViewModel
     {
-        public ChooseACentreViewModel(List<ChooseACentreAccountViewModel> centreUserDetails, string? returnUrl)
+        public ChooseACentreViewModel(
+            List<ChooseACentreAccountViewModel> centreUserDetails,
+            string? returnUrl,
+            bool primaryEmailIsVerified,
+            List<(int centreId, string centreName, string? centreSpecificEmail)> unverifiedCentreEmails
+        )
         {
             CentreUserDetails = centreUserDetails;
             ReturnUrl = returnUrl;
+            PrimaryEmailIsVerified = primaryEmailIsVerified;
+            UnverifiedCentreEmails = unverifiedCentreEmails;
         }
 
         public List<ChooseACentreAccountViewModel> CentreUserDetails { get; set; }
         public string? ReturnUrl { get; set; }
+        public bool PrimaryEmailIsVerified { get; set; }
+        public List<(int centreId, string centreName, string? centreSpecificEmail)> UnverifiedCentreEmails { get; set; }

```

Comment:
The reason I have it here, is because I need to pass it to the VerifyEmailWarningIfAppropriateViewComponent so we can count the number of unverified emails and pluralise the language accordingly. We could just count the number of unverified emails from the CentreUserDetails, but this would require having a separate view component for the Choose a Centre page, since the my Account page still needs the emails themselves to be passed in. I could split them into separate view components, but they have a lot of common code, and the names would become quite long, like ChooseACentreVerifyEmailWarningIfAppropriateViewComponent. What do you think?

Reply:
Ah okay, missed that the callout would depend on the number of unverified centre emails.